app/modules/tls.c

```diff
@@ -579,7 +579,6 @@ static int tls_cert_verify(lua_State *L)
   if (lua_type(L, 1) == LUA_TSTRING) {
     const char *types[2] = { "CERTIFICATE", NULL };
     const char *names[1] = { "certificate" };
-
```

Comment:
Spurious?

Reply:
Removed this to align the code for `tls_cert_verify` and `tls_cert_auth`